lib/routes/defaults.js

```diff
@@ -28,36 +28,44 @@ try {
 
 module.exports = (log, db) => {
 
-  function versionHandler(request, reply) {
+
+ async function versionHandler(request, h) {
```

Comment:
Shouldn't this line be indented 2 spaces?

Reply:
Can we come up with a better name for `h`?

Reply:
> Can we come up with a better name for `h`?

Even though `h` is the name that hapi uses (as per https://github.com/mozilla/fxa-auth-server/pull/2486#discussion_r200157712) I don't think that means we have to use it. It seems like a really bad name to me. If they say it stands for `hapi` there, is there anything stopping us  from calling it `hapi` over here? I think that would make the code much clearer for people reading it.

Reply:
(I genuinely assumed it stood for `handler` when I first read it)

Reply:
(and it's not just for readability, if you ever need to do `git grep h` you'll be pleased if we change that to `hapi`)